sympy/solvers/bivariate.py

```diff
@@ -165,12 +165,24 @@ def _lambert(eq, x):
     # different k values.
     lambert_real_branches = [-1, 0]
     sol = []
-    xe = Dummy('xe')
+
+    # solution of the given Lambert equation is like
+    # sol = -c/b + (a/d)*LambertW(arg, k),
+    # where arg = d/(a*b)*exp((c*d-b*f)/a/b) and k in lambert_real_branches
+    # arg can have more than one values as exp(num/p*den)[exp((c*d-b*f)/a/b)]
```

Comment:
With this notation it is not clear what you are saying. I was treating `exp((x*d-b*f)/a/b)` like `exp(num/(p*den))` and handling the `p` roots of `exp(num/den)`.

Reply:
Okay I will change it if you have a good suggestion let me know.

Reply:
Is it okay now? If it is then I think there is nothing left to improve.